aqua/qt53/Portfile

```diff
@@ -0,0 +1,1775 @@
+            rmd160  [lindex [lindex ${module_info} 0] 0] \
+            sha256  [lindex [lindex ${module_info} 0] 1] \
+            size    [lindex [lindex ${module_info} 0] 2]
+
+        set license_string [lindex ${module_info} 8]
+        set license_string [string range ${license_string} 1+[string first ":" ${license_string}] end]
+        set license_string [string trim  ${license_string}]
+
+        if { ${license_string} ne "" } {
+            license {*}${license_string}
+        }
+
+        foreach deps [lindex ${module_info} 1] {
+            depends_build-append [subst ${deps}]
+        }
+
+        foreach deps [lindex ${module_info} 2] {
+            depends_lib-append [subst ${deps}]
+        }
+
+        foreach qtdeps [lindex ${module_info} 3] {
+            if { ${qtdeps} ne "qtbase" } {
+                depends_lib-append port:${name}-${qtdeps}
+            }
+        }
+
+        description       Tools and Module(s) for Qt Tool Kit ${qt_major}
+
+        set modules_provided_list [lindex ${module_info} 4]
+        if { [llength ${modules_provided_list}] == 1 } {
+            set modules_provided_join [lindex ${modules_provided_list} 0]
+        } elseif { [llength ${modules_provided_list}] == 2 } {
+            set modules_provided_join [join ${modules_provided_list} " and "]
+        } else {
+            set modules_provided_join [join [list [join [lrange ${modules_provided_list} 0 end-1] ", "] [lindex ${modules_provided_list} end]] ", and "]
+        }
+
+        long_description  "Tools and Module(s) for Qt Tool Kit ${qt_major}: ${modules_provided_join}"
+
+        if { ${module} eq "qtbase" } {
+            # this subport uses configure script (NOT qmake)
+            PortGroup           qt5 1.0
+
+            if { ${os.platform} ne "darwin" } {
+                pre-fetch {
+                    ui_warn "${subport} is untested on \"${os.platform}\"."
+                }
+            }
+
+            if { [variant_isset universal] } {
+                pre-fetch {
+                    ui_warn "Multiple architectures is not a Reference Configuration for Qt."
+                    ui_warn "See https://doc.qt.io/qt-5/supported-platforms.html#reference-configurations"
+                }
+            } else {
+                if { ${configure.build_arch} eq "i386" } {
+                    pre-fetch {
+                        ui_warn "32-bit mode is not a Reference Configuration for Qt."
+                        ui_warn "See https://doc.qt.io/qt-5/supported-platforms.html#reference-configurations"
+                    }
+                }
+            }
+
+            # Qt uses TLS protocols introduced in OS X 10.8
+            # see https://developer.apple.com/reference/security/sslprotocol/ktlsprotocol11
+            # see https://developer.apple.com/reference/security/sslprotocol?language=objc
+            pre-fetch {
+                if { ${os.major} < 12 && ![variant_isset openssl] } {
```

Comment:
For consistency, I tweaked qt55 and qt56 to match this (see https://github.com/macports/macports-ports/commit/d9dd27439cf751025b9128cf0b5aff313aae2309).